pkg/logging/logger.go

```diff
@@ -0,0 +1,60 @@
+package logging
+
+import (
+        "io"
+        stdlog "log"
+        "os"
+        "path"
+        "strings"
+
+        log "github.com/sirupsen/logrus"
```

Comment:
please update `Gopkg.toml` for this new dependency

Reply:
Ack